certified-connectors/document-ai-konfuzio/apiDefinition.swagger.json

```diff
@@ -0,0 +1,623 @@
+          "type": "object"
+        },
+        "updated_at": {
+          "title": "Updated at",
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true
+        }
+      }
+    },
+    "SequenceAnnotation": {
+      "required": [
+        "label"
+      ],
+      "type": "object",
+      "properties": {
+        "id": {
+          "title": "ID",
+          "type": "integer",
+          "readOnly": true
+        },
+        "label": {
+          "title": "Label",
+          "type": "integer"
+        },
+        "label_text": {
+          "title": "Label text",
+          "type": "string",
+          "readOnly": true
+        },
+        "label_data_type": {
+          "title": "Label data type",
+          "type": "string",
+          "readOnly": true
+        },
+        "label_threshold": {
+          "title": "Label threshold",
+          "type": "string",
+          "readOnly": true
+        },
+        "section_label_text": {
+          "title": "Section label text",
+          "type": "string",
+          "readOnly": true
+        },
+        "section_label_id": {
+          "title": "Section label id",
+          "type": "integer"
+        },
+        "section": {
+          "title": "Section",
+          "type": "integer",
+          "x-nullable": true
+        },
+        "offset_string": {
+          "title": "Offset string",
+          "type": "string",
+          "x-nullable": true
+        },
+        "offset_string_original": {
+          "title": "Original offset string",
+          "type": "string",
+          "x-nullable": true
+        },
+        "translated_string": {
+          "title": "Translated string",
+          "type": "string",
+          "x-nullable": true
+        },
+        "normalized": {
+          "title": "Normalized",
+          "type": "string",
+          "readOnly": true
+        },
+        "start_offset": {
+          "title": "Start offset",
+          "type": "integer",
+          "maximum": 2147483647,
+          "minimum": -2147483648,
+          "x-nullable": true
+        },
+        "end_offset": {
+          "title": "End offset",
+          "type": "integer",
+          "maximum": 2147483647,
+          "minimum": -2147483648,
+          "x-nullable": true
+        },
+        "accuracy": {
+          "title": "Accuracy",
+          "type": "number",
+          "x-nullable": true
+        },
+        "is_correct": {
+          "title": "Correct",
+          "type": "boolean"
+        },
+        "revised": {
+          "title": "Human revised",
+          "type": "boolean"
+        },
+        "created_by": {
+          "title": "Created by",
+          "type": "integer",
+          "x-nullable": true
+        },
+        "revised_by": {
+          "title": "Revised by",
+          "type": "integer",
+          "x-nullable": true
+        },
+        "bbox": {
+          "title": "Bbox",
```

Comment:
@atraining Is it a typo? If not, can you please give us some context on `Bbox` ?

Reply:
By Bbox we refer to the bounding box which refers to the visual area the SequenceAnnotation was extracted from.